docs/zh/document/developer-guide/sermant-xds-service.md

```diff
@@ -328,11 +331,191 @@ XdsLbPolicy getBaseLbPolicyOfService(String serviceName);
   loadBalanceService.getLbPolicyOfCluster("outbound|8080||service-test.default.svc.cluster.local")；
   ```
 
+## 基于xDS协议的流控服务
+
+### 功能介绍
+
+**基于xDS协议的流控服务**允许Sermant对接Istio的控制平面获取Kubernetes Service的流控配置。
+
+> 说明：使用Sermant的xDS流控服务需服务部署在Kubernetes容器环境并运行Istio。 
+
+### 开发示例
+
+本开发示例基于[创建首个插件](README.md)文档中创建的工程，演示插件如何通过Sermant框架提供的xDS流控服务获取服务的流控规则：
+
+1. 在工程中`template/template-plugin`下的`io.sermant.template.TemplateDeclarer` 类中新增变量`xdsFlowControlService`获取Sermant框架提供的xDS流控服务，用于获取服务的流控规则：
+
+   ```java
+   XdsFlowControlService xdsFlowControlService = ServiceManager.getService(XdsCoreService.class).getXdsFlowControlService();
+   ```
+
+2. 获取xDS流控服务之后，可以调用`xdsFlowControlService`提供的API获取服务的流控规则：
+
+   ```java
+   Optional<XdsInstanceCircuitBreakers> circuitBreakersOptional = xdsFlowControlService.getInstanceCircuitBreakers("spring-test", "v1");
+   System.out.println("circuitBreaker : " + circuitBreakersOptional.get().getInterval());
+   ```
+   
+   > 说明：`service-test`必须是Kubernetes Service的名称，v1是Kubernetes Service的集群名称，获取的流控规则通过Istio提供的[DestinationRule](https://istio.io/latest/zh/docs/reference/config/networking/destination-rule/)、[VirtualService](https://istio.io/latest/zh/docs/reference/config/networking/virtual-service/)和[EnvoyFilter](https://istio.io/latest/docs/reference/config/networking/envoy-filter/)下发，Sermant具体支持的流控规则字段、支持配置的流控规则和配置模版请参考[基于xDS服务的流控能力](../user-guide/sermant-xds.md#基于xds服务的流控能力)一节。
```

Comment:
DestinationRule link unavailable。check it for all xds doc

Reply:
done